llvm/lib/Object/XCOFFObjectFile.cpp

```diff
@@ -1288,7 +1288,11 @@ Expected<bool> XCOFFSymbolRef::isFunction() const {
   if (CsectAuxRef.getSymbolType() == XCOFF::XTY_LD)
     return true;
 
-  return createError("csect symbol has no valid symbol type.");
+  return createError(
+      "Symbol csect aux entry with index " +
```

Comment:
This message doesn't conform to the LLVM coding standards. Please fix.

Reply:
Hope I am doing right this time.